fdm-app/app/lib/greetings.ts

```diff
@@ -0,0 +1,7 @@
+export function getTimeBasedGreeting(date = new Date()): string {
+    const hours = date.getHours();
+    if (hours < 5) return "Goedenacht";
+    if (hours < 12) return "Goedemorgen";
+    if (hours < 18) return "Goedemiddag";
+    return "Goedenavond";
+}
```

Comment:
this greeting is fixed now?

Reply:
You mean time of day or the language? Yes, the hours are fixed not, but depend on the user local time, not server time. Translations are planned with this issue: #8, but probably not for alpha release